pkg/virt-controller/services/template.go

```diff
@@ -317,6 +317,7 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 	var volumeDevices []k8sv1.VolumeDevice
 	var userId int64 = 0
 	var privileged bool = false
+	var qemuUser int64 = 107
```

Comment:
Can we (re-)use something like in https://github.com/kubevirt/kubevirt/pull/3080 and avoid hardcoded group id?

Reply:
Fair point, thanks! Fixed.

Reply:
Oops. I fixed the tests.

Fixing this is a little odd--this code is executed in the virt-controller container. The group id in question is in a different container. Retrieving the GID might require a fair bit of engineering. Is that really worth it?

Reply:
Maybe a compromise could be sth like leveraging getent? I.e. sth like this:

```bash
getent group qemu | cut -d ':' -f 3
```

Reply:
I guess there's a package in go for that, at least we could get rid of that magic number here. Praying the GID will stay stable ;-)

Reply:
The qemu user isn't present in virt-controller's container, so there's nothing to look up. It would require looking up the user info from inside virt-launcher. that would probably require firing up a dummy container.

The GID *is* stable, fwiw. On every redhat-derived system I'm aware of, it's always 107. iirc libvirt-qemu on debian is 64055, which is the closest analogy afaik.

It's really easy to over-engineer this.

Reply:
oh, that's bad...
IIRC the context of the other PR was that he was rebuilding KubeVirt on Suse Linux. So having this hardcoded here will break his build again...
Getting the actual GID from virt-launcher might be overengineering for this edge usecase indeed, but maybe it's easy to make it at least configurable here?

Reply:
Added a config option.

Reply:
thanks! :+1: